css/index.css

```diff
@@ -1,3 +1,359 @@
+        justify-content: center;
+    }
+        header .logo {
+        border: 2px solid lightcoral;
+        display: inline-flex;
+        justify-content: center;
+        margin-top: 25px;
+        order: -1;
+        width: 183px;
+    }
+    header {
+        
+        display: inline-flex;
+        flex-direction: column;
+        justify-content: right;
+    }
+    header nav {
+        width: 100%;
+    }
+}
+
+@media (max-width: 500px) {
+    header {
+        /* border: 1px solid lawngreen; */
+        justify-content: center;
+        width: 100%;
+    }
+    header nav {
+        display: flex;
+        width: 100%;
+        /* border: 1px solid lightcoral; */
+        flex-direction: column;
+        
+    }
+    header nav a {
+        justify-content: center;
+        margin: 15px 0px;
+        border-bottom: 1px solid black;
+        padding: 15px 0px;
+    }
+}
+
+/* cta */
+
+.cta {
+    /* border: 2px solid lightblue; */
+    display: inline-flex;
+    justify-content: center;
+    width: 100%;
+}
+
+.cta .cta-text {
+    /* border: 2px solid lightcoral; */
+  display: inline-flex;
+  flex-direction: column;
+  /* justify-content: space-between; */
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-flex;
+  justify-content: right;
+  margin-top: 69px;
+}
+
+.main-content {
+    display: flex;
+    flex-direction: column;
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+	display: flex;
+  	padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: flex;
+  flex-direction: column;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+    display: inline-flex;
+    /* border: 2px solid lightblue; */
+    margin-top: 37px;
+    padding: 0 10px;
+    justify-content: space-between;
+}
+
+.main-content .bottom-content .text-content {
+  width: 25%;
+  display: inline-flex;
+  
+  flex-direction: column;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+
+@media (max-width: 800px){
+    .container {
+        width: 100%;
+    }
+    .cta img {
+        display: none;
+    }
+    .middle-img {
+        width: 95%;
+        justify-content: center;
```

Comment:
this is interesting to me because you don't have display:flex on this component. justify-content only works within flex. also remember that you should be adding display:flex to the parent component, not the child.